### PR TITLE
Remove: AntoineAflalo.SoundSwitch version 5.6.0.27635

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/26452)

### CODE REVIEW

File: manifests/a/AntoineAflalo/SoundSwitch/5.6.0.27635/AntoineAflalo.SoundSwitch.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PackageIdentifier: AntoineAflalo.SoundSwitch
-PackageVersion: 5.6.0
+PackageVersion: 5.6.0.27635
 PackageName: SoundSwitch
 Publisher: AntoineAflalo
 PackageUrl: https://soundswitch.aaflalo.me/
```
Comment (ItzLevvie) [inline manifests/a/AntoineAflalo/SoundSwitch/5.6.0.27635/AntoineAflalo.SoundSwitch.yaml:5]:
```suggestion
#PackageUrl: https://soundswitch.aaflalo.me/
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by vedantmgoyal2009** [comment]
The URL is working, but don't know why the pipeline is having issues. 🤷

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Hitting a dependency issue.

Dependencies are not yet supported by Windows Package Manager.

**Comment by vedantmgoyal2009** [comment]
How did #16709 get passed?

**Comment by ItzLevvie** [comment]
> How did #16709 get passed?

5.8.1.42207 doesn't have any dependency issues and sometimes the pipelines may not always catch issues like this.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You can remove the SoundSwitch packages that have the `Validation-Unattended-Failed` label as these packages were created before automated VMs were added.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**